client_code/forms/routes/ballot/Index/__init__.py

```diff
@@ -24,10 +24,30 @@ def __init__(self, **properties):
         self.init_components(**properties)
         self.init_tabulator()
 
+    def search_ballot(self, ballot_id):
+        ballot = session.LOCAL_STORE.get(Ballot, ballot_id)
+        if not ballot:
+            ballot = session.sync_ballot(
+                session.SCROLL_STORE, session.LOCAL_STORE, ballot_id
+            )
+        if ballot:
+            form = Read(ballot=ballot)
+            anvil.get_open_form().show_detail(form)
+        else:
+            anvil.Notification(
+                message="The ballot you are looking for does not exist",
+                title="Ballot not found",
+                icon="fa:exclamation-triangle",
+                style="warning",
+            ).show()
+            routing.set_url_hash("", load_from_cache=False)
+
     def show_ballot(self, uuid):
-        ballot = session.LOCAL_STORE.get(Ballot, uuid) or session.sync_ballot(
-            session.SCROLL_STORE, session.LOCAL_STORE, uuid
-        )
+        ballot = session.LOCAL_STORE.get(Ballot, uuid)
+        if not ballot:
+            ballot = session.sync_ballot(
+                session.SCROLL_STORE, session.LOCAL_STORE, uuid
+            )
```

Comment:
Sweep: this code should not have been edited.
"Why has this change been made? It doesn't appear to add anything nor have any connection to https://github.com/electora-vote/electora-app/issues/86"